src/main/kotlin/racing/domain/car/Car.kt

```diff
@@ -0,0 +1,46 @@
+package racing.domain.car
+
+import racing.domain.exception.CarNameLengthExcessException
+
+data class Car(
+    private val _carName: String,
+    private var _moveCount: Int = 0
+) {
+    private var _positionByRound: MutableMap<Int, Int> = mutableMapOf()
```

Comment:
참고로 List, Map 등의 컬렉션 타입의 경우 var로 정의하기보다 mutable 타입에서 활용될 수 있는 add, remove 등으로 조작하는걸 지향하는게 좋습니다! 내부에서 새로운 컬랙션 인스턴스를 생성하는 것보다 적은 리소스를 사용합니다.
```suggestion
    private val _positionByRound: MutableMap<Int, Int> = mutableMapOf()
```

Reply:
네 이건 제가 놓쳤던 부분이네요.
컬렉션은 불변으로 두고, 컬렉션 내부만 mutable하게 가져가는 것이 맞아보입니다.